### ISSUE TITLE
Track CPU, memory & request time

### ISSUE THREAD

**Issue by skreutzberger** [SwiftyBeaver/SwiftyBeaver-Vapor#2]
Vapor itself still needs some more work to allow that but once it is done it would be nice to have.

**Comment by skreutzberger** [comment]
CPU, memory, etc. could be tracked by a “tracker” that is running on an own thread. On the server, New Relic is doing that with a client application / demon AFAIR but for Swift it could be done while being embedded in the application. Let me give some thoughts about that, it should be easy to build with SwiftyBeaver because it is anyway “living” in your application. So tracking of CPU, memory and maybe also HTTP request metrics should be easy.

discussed with vdka and loganwright on Vapor Slack channel
